Akinator.Api.Net/Utils/ServerSelector.cs

```diff
@@ -29,7 +29,7 @@ public static class ServerSelector
                 {
                     Language.Arabic, new[]
                     {
-                        new Server(ServerType.Person, "srv12.akinator.com:9187/ws")
+                        new Server(ServerType.Person, "srv2.akinator.com:9155/ws")
```

Comment:
Why did you change this? Did the server change or was it the wrong type? It worked yesterday for me.

Reply:
because this server contain limited characters and doesn't work in every country (doesn't work for me in Egypt (it works when I'm using vpn))